sdcm/cluster_k8s/__init__.py

```diff
@@ -333,6 +330,14 @@ def soft_reboot(self):
     # On kubernetes there is no stop/start, closest analog of node restart would be soft_restart
     restart = soft_reboot
 
+    @cached_property
+    def public_ip_address(self):
+        return super().public_ip_address
+
+    @cached_property
+    def private_ip_address(self):
+        return super().public_ip_address
+
```

Comment:
How we can update these addresses if they will be changed?

Is it intended that private_ip_address calls `super().public_ip_address` ?

Reply:
redone, please check